src/less/variables/ds6/icon-variables.less

```diff
@@ -290,3 +290,6 @@
 @icon-checkbox-checked-disabled-small-base64: "PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGlkPSJpY29uLWNoZWNrYm94LWNoZWNrZWQtc21hbGwiIHZpZXdCb3g9IjAgMCAxNCAxNCI+CjxwYXRoIGQ9Ik0xMyAwSDFhMSAxIDAgMDAtMSAxdjEyYTEgMSAwIDAwMSAxaDEyYTEgMSAwIDAwMS0xVjFhMSAxIDAgMDAtMS0xem0tMi4yOSA1LjcxbC00IDRhMSAxIDAgMDEtMS40MSAwbC0yLTJhMSAxIDAgMDExLjQxLTEuNDJMNiA3LjU5IDkuMjkgNC4zYTEgMSAwIDAxMS40MSAxLjQxaC4wMXoiLz4KPC9zdmc+";
 @icon-radio-checked-disabled-base64: "PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGlkPSJpY29uLXJhZGlvLWNoZWNrZWQiIHZpZXdCb3g9IjAgMCAyNCAyNCI+CjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgZD0iTTEyIDBDNS4zNzMgMCAwIDUuMzczIDAgMTJzNS4zNzMgMTIgMTIgMTIgMTItNS4zNzMgMTItMTJBMTEuOTk4IDExLjk5OCAwIDAwMTIgMHptMCAyMkM2LjQ3NyAyMiAyIDE3LjUyMyAyIDEyUzYuNDc3IDIgMTIgMnMxMCA0LjQ3NyAxMCAxMC00LjQ3NyAxMC0xMCAxMHptNy0xMGE3IDcgMCAxMS0xNCAwIDcgNyAwIDAxMTQgMHoiLz4KPC9zdmc+";
 @icon-radio-checked-disabled-small-base64: "PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGlkPSJpY29uLXJhZGlvLWNoZWNrZWQtc21hbGwiIHZpZXdCb3g9IjAgMCAxNiAxNiI+CjxwYXRoIGQ9Ik04IDBhOCA4IDAgMTEwIDE2QTggOCAwIDAxOCAwem0wIDJhNiA2IDAgMTAwIDEyQTYgNiAwIDAwOCAyem0wIDJhNCA0IDAgMTEwIDggNCA0IDAgMDEwLTh6Ii8+Cjwvc3ZnPg==";
+
+@icon-information-hover-color: #3665f3;
```

Comment:
You can use the variable `@color-b4`

Reply:
And I would remove those colors from this file as well and keep have them in tooltip-variables.less file
OR
Just use the color directly (the `@color-b4`) in the tooltip.less file since there aren't any tooltip variable files.